README.md

```diff
@@ -0,0 +1,42 @@
+# ereolengo.dk
+
+Modules for ereolengo.dk.
+
+## Coding standards
+
+All code must adhere to [the Drupal Coding
+Standards](https://www.drupal.org/docs/develop/standards).
+
+Install tools to check coding standards:
+
+```sh
+docker run --rm -it --volume $(pwd):/app itkdev/php7.0-fpm:latest composer2 install
```

Comment:
Why using `docker run`? Do the project come with docker-compose.yml? 

This could be `docker-compose run phpfpm composer2 install` which would ensure right php version and no need to update dokumentation with php version opgrade.

Reply:
We don't have `docker-composer.yml` in https://github.com/eReolen/ereolengo – only in https://github.com/eReolen/base.